engines/ospray/OSPRayMaterial.cpp

```diff
@@ -136,10 +134,10 @@ OSPTexture OSPRayMaterial::_createOSPTexture2D(Texture2DPtr texture)
 
     OSPTexture ospTexture = ospNewTexture("texture2d");
 
-    const osp::vec2i size{int(texture->getWidth()), int(texture->getHeight())};
+    const Vector2i size{int(texture->getWidth()), int(texture->getHeight())};
 
-    ospSet1i(ospTexture, "type", static_cast<int>(type));
-    ospSet2i(ospTexture, "size", size.x, size.y);
+    osphelper::set(ospTexture, "type", static_cast<int>(type));
+    osphelper::set(ospTexture, "size", size);
```

Comment:
Could this work:

    osphelper::set(ospTexture, "size", {int(texture->getWidth()), int(texture->getHeight())});

Reply:
No